SecondProgram.cs

```diff
@@ -0,0 +1,191 @@
+
+
+/*
+
+Created By Ragu Balagi Karuppaiah.
+
+Date : 22.06.2019
+
+
+Solution For Second Assignment Second Problem.(Tavisca)
+
+
+
+*/
+
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SecondProblem
+{
+    public class ForumPostEasy
+    {
+        string PostTime;
+        
+        public string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
+            
+            if(exactPostTime.Length > 1)
+            {
+                for (int i = 0; i < exactPostTime.Length; i++)
+                {
+                       for(int j=i+1; j<exactPostTime.Length; j++)
+                    {
+                        if(exactPostTime[i].Equals(exactPostTime[j]))
+                        {
+                            if(!showPostTime[i].Equals(showPostTime[j]))
+                            {
+                                return "impossible";
+                            }
+                        }
+                    }
+                }
+            
+            string[] results = new string[exactPostTime.Length];
+            int[] sum = new int[exactPostTime.Length];
+            int mintime = 0;
+            string time;
+            for(int i =0; i< exactPostTime.Length; i++)
+            {
+                //Console.WriteLine(exactPostTime[i]);
+
+                int hr = int.Parse(exactPostTime[i].Split(':')[0]);
+                int min = int.Parse(exactPostTime[i].Split(':')[1]);
+                int sec = int.Parse(exactPostTime[i].Split(':')[2]);

```

Comment:
What if the exactPostTime[i].Split(':'[2]) is null??
Use int.Tryparse instead of only parse.